components/dashboards-web-component/src/designer/DashboardDesigner.jsx

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *  Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Please remove this space. It's not needed

Reply:
it is requested by SajithAr